content/github/creating-cloning-and-archiving-repositories/about-code-owners.md

```diff
@@ -83,6 +83,13 @@ apps/ @octocat
 # subdirectories.
 /docs/ @doctocat
 ```
+#### Syntax exceptions
+There are some syntax rules for .gitignore files that do not work in CODEOWNERS files:
```

Comment:
Previously we didn't use the dot in `gitignore` so I suggest removing it here for consistency.
```suggestion
There are some syntax rules for gitignore files that do not work in CODEOWNERS files:
```